util/wordpress.js

```diff
@@ -268,17 +269,24 @@ function setupTheme(generator, config, done) {
 	if (fs.existsSync(themePackageJson)) {
 		var oldDir = process.cwd();
 		process.chdir(themePath);
-		console.log(chalk.green('Installing Node Packages (be patient)'));
+		console.log(chalk.green('Installing NPM Packages (be patient)'));
 
 		exec('npm install', function(err, stdout, stderr) {
 			if (err) {
-				console.error('Error installing packages', err, stdout, stderr);
+				console.error('Error installing NPM packages', err, stdout, stderr);
 				return done();
 			}
-
+			if (config.bower && fs.existsSync(BowerFile)) {
+				exec('bower install', function(err, stdout, sterr) {
```

Comment:
This will have to be refactored.  If `config.bower` is true, it will result in running the lines at 287 before the bower install is complete.  Probably needs flow control logic, maybe look at using https://www.npmjs.com/package/run-series?